Docs/beta/snippets/torch/export.py

```diff
@@ -34,8 +34,9 @@
 #
 #  @@-COPYRIGHT-END-@@
 # =============================================================================
-# pylint: disable=all
-# Export the model
-# Export the model which saves pytorch model without any simulation nodes and saves encodings file for both
-# activations and parameters in JSON format
-model.export(path='./', filename_prefix='<name_prefix>', dummy_input=dummy_input.cpu())
+# pylint: disable=missing-docstring
+
+    # Export the model
+    # Export the model which saves pytorch model without any simulation nodes and saves encodings file for both
+    # activations and parameters in JSON format
+    model.export(path='./', filename_prefix='<name_prefix>', dummy_input=dummy_input.cpu())
```

Comment:
Same comment as above regarding the indentation

Reply:
Same.